test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -803,6 +886,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.UserAgent | Should Match $Pattern
     }
 
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"{
+        $command = 'Invoke-WebRequest https://prod.idrix.eu/secure/'
```

Comment:
@TravisEz13 unfortunately, I'm a novice with Docker and have no experience with ASP.NET. I could look into it, but It will take a considerable amount of time to figure out what all needs to be done. I'm not opposed to looking into it, but, perhaps there is some way to engage someone else for help on this? I'm not really sure how to proceed.